ci_scripts/ci_pre_xcodebuild.sh

```
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+cd ../Ruddarr/ || exit 1
+
+plutil -replace APNsKey -string "$APNS_SECRET" Info.plist
+plutil -insert CI_BRANCH -string "$CI_BRANCH" Info.plist
```

We should fail if these ENV vars are not set.